dashboard/config/course_offerings/jigsaw.json

```diff
@@ -9,7 +9,7 @@
   "grade_levels": "K,1,2",
   "header": null,
   "image": "https://images.code.org/8fb543a9ed9f0fe3ed538e6d1bf5e189-Jigsaw2.png",
-  "cs_topic": null,
+  "cs_topic": "digital_literacy",
```

Comment:
I'm not super familiar with the best way to add this, but I'm under the impression that there's a 2-way sync between JSONs in levelbuilder and JSONs in prod?